cis/gcp/compute/gcp-cis-4.3-compute-ensure-oslogin-is-enabled-for-a-project/test/gcp-cis-4.3-compute-ensure-oslogin-is-enabled-for-a-project/failure-null-metadata.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This Sentinel configuration file should not be required if you implement my suggestion.